cmd/snap/cmd_sign.go

```diff
@@ -47,7 +51,12 @@ func init() {
 	}, map[string]string{
 		// TRANSLATORS: This should not start with a lowercase letter.
 		"k": i18n.G("Name of the key to use, otherwise use the default key"),
-	}, nil)
+	}, []argDesc{{
+		// TRANSLATORS: This needs to begin with < and end with >
+		name: i18n.G("<filename>"),
+		// TRANSLATORS: This should not start with a lowercase letter.
+		desc: i18n.G("Filename instead of stdio"),
```

Comment:
It's `stdin`, not `stdio`. I think i'd rather `File to sign (defaults to stdin)`